server.js

```diff
@@ -1,6 +1,9 @@
+require('dotenv').config();
 const express = require("express");
 const mongoose = require("mongoose");
 const app = express();
+
+const shoeRoutes = require("./routes/laces.routes");
```

Comment:
Why are you importing shoe routes from file laces routes?
And why do you have two const variables with the same name?